src/components/Card/ProductCard/ProductCard.jsx

```diff
@@ -1,15 +1,20 @@
+import { Link } from "react-router-dom";
+import { useFilter } from "../../../contexts/filterContext/context";
```

Comment:
I think you can add `index.js` in contexts folder for better management of import and export.